### PR TITLE
add message if no active reservation is found

### PR DESCRIPTION
**Pull request by craigcooper11**

### CODE REVIEW

File: pkg/reservations/base.go [hunk 1]
```diff
@@ -98,16 +99,18 @@ func (w *TextWriter) WriteAll(out io.Writer, rez []TZReservation) error {
 	return nil
 }
 
-func (w *TextWriter) WriteFilter(out io.Writer, rez []TZReservation, filter Filter) error {
+func (w *TextWriter) WriteFilter(out io.Writer, rez []TZReservation, filter Filter) (error, int) {
```
Comment (nathanagood) [inline pkg/reservations/base.go:102]: Idiomatically, flip the tuple to be `(int, error)` as `error`, `exists`, `ok`, etc. always seems to be second in the tuple. Great idea to use this, BTW.

**Review by nathanagood — CHANGES_REQUESTED** [review]
Good stuff, love the addition of the match count in the tuple. Idiomatically, as mentioned, I recommend just flipping them.

File: pkg/reservations/base.go (hunk 1 above)
Comment (craigcooper11) [reply pkg/reservations/base.go]: Sounds good! Made that change in the latest commit.

**Review by nathanagood — APPROVED** [review]
LGTM

**Pull request merged**